src/gui/widgets/SimpleTextFloat.cpp

```diff
@@ -28,6 +28,7 @@
 #include <QStyleOption>
 #include <QHBoxLayout>
 #include <QLabel>
+#include <QTimer>
```

Comment:
Unfortunately, this now includes `QTimer` twice as it is already included at the top of the includes. 😅

Can you please fix this last thing and then the PR is ready to go?